src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java

```diff
@@ -6,13 +6,15 @@
     private String userEmail;
     private String login;
     private String password;
+    private int amountOfDays;
 
     public GitAuthData(String... args) {
         repositoryName = args[0];
         userFullName = args[1];
         userEmail = args[2];
         login = args[3];
         password = args[4];
+        amountOfDays = Integer.parseInt(args[5]);
```

Comment:
The `amountOfDays` is not a part of `AuthData`. Please remove it out of here.

Reply:
Done